src/bun.js/bindings/ZigSourceProvider.cpp

```diff
@@ -153,6 +156,9 @@ void SourceProvider::cacheBytecode(const BytecodeCacheGenerator& generator)
     if (update)
         m_cachedBytecode->addGlobalUpdate(*update);
 }
+SourceProvider::~SourceProvider()
+{
```

Comment:
is the remove of the two calls in the destructor intentional?

Reply:
The two calls do nothing so it's just confusing